app/views/templates/MainTemplate.scala.html

```diff
@@ -71,7 +73,7 @@
     userResearchBannerUrl = if (renderConfig.showUserResearchBanner) Some(renderConfig.userResearchUrl) else None,
     serviceName = Some(messages("site.service_name")),
     serviceUrl = Some(appConfig.serviceUrl),
-    signOutUrl = if (canSignOut) Some(renderConfig.signOutUrl) else None,
+    signOutUrl = if (canSignOut) Some(lrn.map(x => routes.DeleteLockController.delete(x).url).getOrElse(renderConfig.signOutUrl)) else None,
```

Comment:
Very minor refactor but this could be `lrn.map(routes.DeleteLockController.delete(_).url)`